docassemble/ALWeaver/test_docx_files.py

```diff
@@ -1,14 +1,35 @@
 import unittest
 from .interview_generator import (
-    DAField,
+    DAFieldList,
     get_docx_variables,
     is_reserved_docx_label,
     get_pdf_variable_name_matches,
 )
-from docassemble.base.util import DAFile
+from .validate_template_files import matching_reserved_names
+from docassemble.base.util import DAStaticFile
 from docx2python import docx2python
 from pathlib import Path
 
+import docassemble.base.functions
+   
+class MockDAStaticFile(DAStaticFile):
```

Comment:
Haha, this was a quick and dirty class. If we want to start using in a third file after this, should probably make a separate test utilities module and clean it up a bit.

Reply:
Agreed, but I think for another PR